docs/src/docs/next-steps.md

```diff
@@ -179,13 +225,15 @@ kit inspect mymodel:challenger
 }
 ```
 
-## Tag ModelKits and keep your registry tidy
+`size` is shown in kilobytes (KB). For more information on the manifest can be found in the [specification documentation](./modelkit/spec.md).
```

Comment:
It's actually raw bytes (divide by 1,000 for KB, 1,000,000 for MB)

Reply:
fixed in latest commit